src/components/TextField.vue

```diff
@@ -29,6 +30,12 @@
       :for="id"
     >
       {{ label }}
+      <span
+        v-if="required"
+        class="text-field__required-text"
+      >
+        (required)
```

Comment:
Would you mind making this translatable? This would mean adding something like this to `lang/en.js`:

```js
{
  "general": {
    "required": "(required)"
  }
}
```

Then do this:
```suggestion
        {{ $t('general.required') }}
```